### PR TITLE
[Site] Updated Used By Section

### PR DESCRIPTION
**Pull request by NikhilSharma03**
**Description**
Updated the Used By Section style as given in mockup design.

## Changes Made
- Updated Companies size and color
- Updated Buttons color
- Updated header font sizes

## After Changes
Screenshot from 2021-07-03 14-15-07

**Notes for Reviewers**

Let me know if there are any other changes we can make.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -283,9 +283,9 @@ <h1 class="used-by">Used By</h1>
 	<section class="adopter-section">
 		<div class="adopter-container">
 			<h3 class="adopter-title">Add yourself as an <strong>adopter</strong></h3>
-			<p class="adopter-description">and share your service mesh story with the community</p>
+			<p class="adopter-description">Share your service mesh story with the community</p>
```
Comment (leecalcote) [inline index.html:286]: For the time being, let's cherrypick this text update and proceed, while leaving the other CSS updates to the side for the time being.

The grayscale on the logos is intentional. The new design (in Figma) isn't necessarily better.

**Comment by leecalcote** [comment]
@NikhilSharma03 I appreciate your effort here.

File: index.html (hunk 1 above)
Comment (NikhilSharma03) [reply index.html:286]: Yeah Sure. I will undo the other changes.

**Comment by NikhilSharma03** [comment]
> @NikhilSharma03 I appreciate your effort here.

Thank you. Will do my best on the next changes.

**Comment by NikhilSharma03** [comment]
Except for the text update, I undid the previous adjustments

**Comment by NikhilSharma03** [comment]
@leecalcote I have undone the previous adjustments except for the text content. Sorry for the trouble.

**Pull request closed without merging**